### PR TITLE
docs: Add helm3-dependency example demonstrating customization of OTS chart using Helm 3

### PR DESCRIPTION
**Pull request by ksimka**
I tried to use OTS chart in Argo following the helm-dependency example and spent some time to find out why Argo uses helm2 for my chart instead of v3 (my subchart was made for helm 3). And finally it turned out you must provide a helm 3 compatible Chart.yaml in the root directory instead of only "name" (like in example) and write dependencies there instead of requirements.yaml file.

### CODE REVIEW

**Comment by darkdragn** [comment]
@ksimka I was just flipping through, sorry I didn't have the chance to just take care of this myself, since I'm on my phone. 

I noticed that the Chart.yaml looks great, with dependencies listed in it, but the README wasn't updated to remove the reference to requirements.txt. It still contains a section mentioning the requirements.txt containing dependencies, instead of stating it's been migrated to Chart.yaml.

I would also appreciate if you mention in the README that this will __only__ work if the Chart.yaml contains `apiVersion: v2` due to the way argocd detects which helm version to use. If someone just accidentally forgets the apiVerions key, it'll attempt helm2, and I feel like that is a pitfall someone just getting started with argocd might fall into.

### Bottom Line

- Remove reference to requirements.txt, change README to ref dependencies in Chart.yaml
- Add mention of helm 3 only used when Chart.yaml includes `apiVersion: v2`, otherwise dependencies in Chart.yaml is ignored.

**Comment by ksimka** [comment]
I'm sorry, I can't get it work anymore. Wordpress chart in given repository is deprecated, and it's recommended to use bitnami's chart, but their chart uses v1 too. I spent some time trying to use it with Helm 3 in Argo, but no success. I'm afraid I don't have enough time for this, so it's better to close this PR I think.

**Comment by darkdragn** [comment]
Or I could take it over, no worries, man. I understand being busy, believe me, is 10 years in the navy taught me anything, it's that, lol
